scripts/e2e-test.sh

```diff
@@ -16,7 +16,9 @@ TOPDIR=$(realpath "$SCRIPTDIR/..")
 #   2. ms_pod_disruption SHOULD be the last test before uninstall
 #
 DEFAULT_TESTS="install basic_volume_io csi resource_check replica rebuild ms_pod_disruption uninstall"
-ONDEMAND_TESTS="install basic_volume_io csi resource_check uninstall"
+#ONDEMAND_TESTS="install basic_volume_io csi resource_check uninstall"
+#temporary 
```

Comment:
Can this be removed?

Reply:
That was temporary I wanted to do a CI run so that we can get loki logs that I can point to in the bug I will raise. This file has now been restored.
This test will always fail at the moment so the it will not be included into CI runs.